Gemfile

```diff
@@ -34,6 +34,7 @@ if RUBY_ENGINE == "ruby"
   gem 'bluecloth'
   gem 'rdiscount'
   gem 'RedCloth'
+  gem "activesupport", "~> 5.1.6"
```

Comment:
Sorry, I can't make sense why is this needed.

Reply:
activesupport-5.2 breaks a test added by you, because the behavior of `HashWithIndifferentAccess#slice` has been modified.
See: https://travis-ci.org/sinatra/sinatra/builds/372333244
We can also delete the test instead of this patch.

Reply:
Ah, I got it.
`IndifferentHash#slice` test should be skipped on ruby 2.4 and before versions with this line https://github.com/sinatra/sinatra/blob/master/test/indifferent_hash_test.rb#L173

But `rabl` gem using `active_support` which is includes `Hash#slice` core extention method, because of that, These test are not skipped.

Then, in ruby 2.5 tests passing because it references ruby core `Hash#slice` method.

I think that maybe removing `active_support`  dependency is best, but the changes looks good for me.